account_invoice_blocking/tests/test_account_invoice_blocking.py

```diff
@@ -11,25 +11,24 @@ def setUp(self):
         Setup test instances
         """
         super(TestAccountInvoiceBlocking, self).setUp()
-
+        self.account_obj = self.env['account.account']
+        self.invoice_line_obj = self.env['account.invoice.line']
+        self.invoice_obj = self.env['account.invoice']
         company = self.env.ref('base.main_company')
-
         # Instance: account type (receivable)
         self.type_recv = self.env.ref('account.data_account_type_receivable')
-
         # Instance: account type (payable)
         self.type_payable = self.env.ref('account.data_account_type_payable')
-
         # account (receivable)
-        account_recv = self.env['account.account'].create({
+        account_recv = self.account_obj.create({
```

Comment:
Why are you replacing the `env['account.account']` with `account_obj`? There's no really any need of it. And it's not even mentioned in the OCA guidelines.

Same discourse for `invoice_line_obj` and `invoice_obj`.